lib/much-rails/not_given.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require "much-not-given"
+
+module MuchRails; end
+module MuchRails::NotGiven
+  include MuchNotGiven
+end
```

Comment:
This should be aliased just like Boolean above.
```suggestion
module MuchRails
  NotGiven = MuchNotGiven
end
```

Reply:
This doesn't work because `MuchNotGiven` is a mix-in...unless I'm missing something.

Reply:
It shouldn't matter - it is just a constant in this context.